kubernetes/helm_charts/core/apimanager/templates/deployment.yaml

```diff
@@ -52,6 +52,17 @@ spec:
               command:
               - kong
               - quit
+{{- if .Values.enable_apiproxyvalidator }}
```

Comment:
Do we still need these in this chart? Please can you revert them to original state?

Reply:
just for additional check, do i remove them?

Reply:
we don't need them, will remove it right away.

Reply:
removed it.